posthog/api/insight.py

```diff
@@ -468,6 +468,12 @@ def _filter_request(self, request: request.Request, queryset: QuerySet) -> Query
             elif key == "my_last_viewed":
                 if str_to_bool(request.GET["my_last_viewed"]):
                     queryset = self._annotate_with_my_last_viewed_at(queryset).filter(my_last_viewed_at__isnull=False)
+            elif key == "feature_flag":
+                feature_flag = request.GET["feature_flag"]
+                queryset = queryset.filter(
+                    Q(filters__breakdown__icontains=f"$feature/{feature_flag}")
```

Comment:
Bug: (non-blocking, edge case we can probs disregard for now)

Say I have 2 feature flags used in insights:

`experience-continuity` and `experience-continuity-failover` - this query will match both vs just one.

Reply:
yeah i also didn't take into account when the feature flag property is used in an event property either so we can definitely make improvements on this if it gets used often enough 🤗